src/app/controllers/authentication.server.controller.js

```diff
@@ -83,10 +83,19 @@ exports.validateDomain = function (req, res, next) {
   })
 }
 
+/**
+ * Returns the error message when a user cannot perform an action on a form
+ * @param {String} user -  user email
+ * @param {String} title -  form title
+ * @returns {String} - the error message
+ */
+const makeUnauthorizedMessage = (user, title) => {
+  return `User ${user} is not authorized to perform this operation on Form: ${title}.`
```

Comment:
Will this be displayed on the UI in the event of HTTP 403? May I suggest shortening the message since the title can get really long?

```suggestion
  return `User ${user} is not authorized to perform this operation on form.`
```

Reply:
e.g. say, when looking up the user fails during transfer due to network error

Reply:
i think this is a product decision.

Reply:
TODO: test the UI and see if there is an existing glitch

Reply:
Can test if there's a UI glitch with a long title. If it wraps correctly then ok to include

Reply:
![image](https://user-images.githubusercontent.com/5405953/90728243-2859e180-e2f7-11ea-8228-9bf72730d61b.png)

does not wrap

Reply:
should fix in another pr